circle.yml

```diff
@@ -21,8 +21,8 @@ dependencies:
 
 test:
   override:
-    - ./build.sh --test --generate
-    - ./build.sh --test --generate --race
+    - ./build.sh --test --generate --no-vet
+    - ./build.sh --test --generate --race --no-vet
```

Comment:
Ah, okay. I wasn't sure if there was a difference between the way InfluxDB and Kapacitor use that option. I remember seeing quite a few errors when I was testing it out, though.